modules/serverless-rn-1-28-0.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-28-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.28
+
+{ServerlessProductName} 1.28 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-28-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.7.
+* {ServerlessProductName} now uses Knative Eventing 1.7.
+* {ServerlessProductName} now uses Kourier 1.7.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.7.
+* {ServerlessProductName} now uses Knative Kafka 1.7.
+* The `kn func` CLI plug-in now uses `func` FIXME version.
+
+* Node.js and TypeScript runtimes for {ServerlessProductName} Functions are now Generally Available (GA).
+
+* Multi-container support for Knative Serving is now available as a Technology Preview. This feature allows you to use a single Knative service to deploy a multi-container pod.
+
+* In a future release, the following components of Knative Eventing will be scaled down from two pods to one:
++
+--
+* `imc-controller`
+* `imc-dispatcher`
+* `mt-broker-controller`
+* `mt-broker-filter`
+* `mt-broker-ingress`
+--
+
+* The `serverless.openshift.io/enable-secret-informer-filtering` annotation for the Serving CR is now deprecated. The annotation is valid only for Istio, and not for Kourier.
++
+With {ServerlessProductName} 1.28, the {ServerlessOperatorName} allows injecting the environment variable `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID` for both `net-istio` and `net-kourier`.
```

Comment:
Do we document anywhere how to set the ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID env ?  I believe this should be documented as part of https://issues.redhat.com/browse/SRVKS-947 

It seems to me we're now deprecating the annotation but we don't really provide the alternative, as we don't document how to actually do it.

Reply:
Yes, SRVKS-947 is here https://github.com/openshift/openshift-docs/pull/58234